0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -20,11 +20,13 @@ This spec depends on the [ILP spec](../0003-interledger-protocol/).
 | | [**disconnect**](#disconnect) ( ) `⇒ Promise.<null>` |
 | | [**isConnected**](#isconnected) ( ) `⇒ Boolean` |
 | | [**getInfo**](#getinfo) ( ) <code>⇒ Promise.&lt;[LedgerInfo](#class-ledgerinfo)></code> |
+| | [**getPrefix**](#getprefix) ( ) `⇒ Promise<String>` |
+| | [**getAddress**](#getaddress) ( ) `⇒ Promise<String>` |
```

Comment:
What's the thinking behind `getPrefix`? Who would call this and what's the use case? Is it discussed in an existing issue?